mongodbatlas/resource_mongodbatlas_project_invitation.go

```diff
@@ -0,0 +1,270 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+func resourceMongoDBAtlasProjectInvitation() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceMongoDBAtlasProjectInvitationCreate,
+		ReadContext:   resourceMongoDBAtlasProjectInvitationRead,
+		DeleteContext: resourceMongoDBAtlasProjectInvitationDelete,
+		UpdateContext: resourceMongoDBAtlasProjectInvitationUpdate,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceMongoDBAtlasProjectInvitationImportState,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"username": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"invitation_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"expires_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"roles": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
```

Comment:
Same situation as the organization validation

Reply:
Fixed